### ISSUE TITLE
Tool to move docs

### ISSUE THREAD

**Issue by peterbe** [mdn/yari#1379]
1. Finish https://github.com/mdn/yari/pull/1358
2. Move whatever the CRUD toolbar does for redirects deep into Yari's builder rather than the `server`.
3. (if time allows) Edit the Toolbar edit-form so the "Slug" is not part of the form. 
4. (if time allows) In the Toolbar, a dedicated functionality for moving the current doc (will sit on the same core functionality as the redirects CLI does)

**Comment by peterbe** [comment]
This is blocked by https://github.com/mdn/yari/pull/1359

**Comment by escattone** [comment]
@fiji-flo I was looking through the issues and found this old one -- https://github.com/mdn/yari/issues/1051 -- which I thought had some information that you might find interesting.

**Comment by escattone** [comment]
Closed via #1510
